src/SDKs/ServiceBus/ServiceBus.Tests/Tests/ScenarioTests.QueuesTests.Detail.CRUD.cs

```diff
@@ -0,0 +1,165 @@
+
+
+                // Create Queue1
+                var queueName1 = TestUtilities.GenerateName(ServiceBusManagementHelper.QueuesPrefix);
+                var createQueueResponse1 = this.ServiceBusManagementClient.Queues.CreateOrUpdate(resourceGroup, namespaceName, queueName1,
+                new SBQueue() { EnableExpress = true });
+
+
+                // Update Queue. 
+                var updateQueuesParameter = new SBQueue()
+                {
+                    EnableExpress = true,
+                    MaxDeliveryCount = 5,
+                    MaxSizeInMegabytes = 1024,
+                    ForwardTo = queueName1,
+                    ForwardDeadLetteredMessagesTo = queueName1
+                    
+                };
+
+                var updateQueueResponse = ServiceBusManagementClient.Queues.CreateOrUpdate(resourceGroup, namespaceName, queueName, updateQueuesParameter);
+                Assert.NotNull(updateQueueResponse);
+                Assert.True(updateQueueResponse.EnableExpress);
+                Assert.Equal(updateQueueResponse.ForwardTo, queueName1);
+                Assert.Equal(updateQueueResponse.ForwardDeadLetteredMessagesTo, queueName1);
+
+
+                // Update Queue with max AutoDeletOnIdeal. 
+                var updateQueuesADonIParameter = new SBQueue()
+                {
+                    AutoDeleteOnIdle = TimeSpan.MaxValue,
+                    EnableExpress = true,
+                    MaxDeliveryCount = 5,
+                    MaxSizeInMegabytes = 1024,
+                    ForwardTo = queueName1,
+                    ForwardDeadLetteredMessagesTo = queueName1
+                };
+
+                var updateQueueADonIResponse = ServiceBusManagementClient.Queues.CreateOrUpdate(resourceGroup, namespaceName, queueName, updateQueuesADonIParameter);
+                Assert.NotNull(updateQueueADonIResponse);
+                Assert.True(updateQueueADonIResponse.EnableExpress);
+                Assert.Equal(updateQueueADonIResponse.ForwardTo, queueName1);
+                Assert.Equal(updateQueueADonIResponse.ForwardDeadLetteredMessagesTo, queueName1);
+                Assert.Equal(updateQueueADonIResponse.AutoDeleteOnIdle, TimeSpan.MaxValue);
+
+                TestUtilities.Wait(TimeSpan.FromSeconds(10));
+
+                // Update Queue with max DefaultMessageTimeToLive. 
+                var updateQueuesTtlParameter = new SBQueue()
+                {
+                    DefaultMessageTimeToLive = TimeSpan.MaxValue,
+                    EnableExpress = true,
+                    DeadLetteringOnMessageExpiration = true,
+                    MaxDeliveryCount = 5,
+                    MaxSizeInMegabytes = 2048,
+                    ForwardTo = queueName1,
+                    ForwardDeadLetteredMessagesTo = queueName1
+                };
+
+                var updateQueueTtlResponse = ServiceBusManagementClient.Queues.CreateOrUpdate(resourceGroup, namespaceName, queueName, updateQueuesTtlParameter);
```

Comment:
Same comment as above for property value verification

Reply:
Done!